packages/jest-cli/src/pretransform_files.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {Test} from 'types/TestRunner';
+import type {GlobalConfig} from 'types/Config';
+import type {Context} from 'types/Context';
+
+import DependencyResolver from 'jest-resolve-dependencies';
+import workerFarm from 'worker-farm';
```

Comment:
Use the new `jest-worker` for this?

Reply:
that was my first thought, but then i realized that all other parallelizations in `jest-cli` use the old way and decided not to diverge :)